jormungandr/src/main.rs

```diff
@@ -340,16 +340,17 @@ pub struct InitializedNode {
 
 fn initialize_node() -> Result<InitializedNode, start_up::Error> {
     let command_line = CommandLine::load();
-    let raw_settings = RawSettings::load(command_line)?;
 
-    if raw_settings.full_version() {
+    if command_line.full_version {
         println!("{}", env!("FULL_VERSION"));
         std::process::exit(0);
-    } else if raw_settings.source_version() {
+    } else if command_line.source_version {
         println!("{}", env!("SOURCE_VERSION"));
         std::process::exit(0);
     }
 
```

Comment:
These can be removed!
https://github.com/input-output-hk/jormungandr/blob/44c8266329c75cb399c10e65d4344f309e43e030/jormungandr/src/settings/start/mod.rs#L54-L61

Reply:
thanks for spotting that